### PR TITLE
Quell warnings

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

File: tests/test_RFT.cpp [hunk 1]
```diff
@@ -457,8 +457,8 @@ namespace {
 
         explicit Setup(const ::Opm::Deck& deck)
             : es    { deck }
-            , python{ }
             , sched { deck, es , python }
+            , python{ }
```
Comment (bska) [inline tests/test_RFT.cpp]: This change is wrong.  The initialization of `sched` uses the `python` member.  To actually fix this you need to reorder the data members instead.

Comment (akva2) [reply tests/test_RFT.cpp]: indeed. i did not see that, sadly not caught in review.

**Pull request merged**